src/models/connectionStore.ts

```diff
@@ -294,11 +294,23 @@ export class ConnectionStore {
      * Clear all recently used connections from the MRU list.
      */
     public async clearRecentlyUsed(): Promise<void> {
+        // Get all recent connection profiles and delete the associated credentials.
+        const mruList = this.getRecentlyUsedConnections();
+        let errorOccured = false;
+        for (const connection of mruList) {
+            const credentialId = ConnectionStore.formatCredentialId(connection.server, connection.database, connection.user, ConnectionStore.CRED_MRU_USER);
+            try {
+                await this._credentialStore.deleteCredential(credentialId);
+            }
+            catch (err) {
+                errorOccured = true;
+                console.error(LocalizedConstants.deleteCredentialError, credentialId, err);
```

Comment:
It would be better to log this to the output channel instead so users don't have to hunt around for how to open the developer tools (and where in the tools this message would be...) and then include an action in the error message to open the output channel.

Reply:
done. the method i am using will open the output channel directly.